sale_comment_template/__manifest__.py

```diff
@@ -20,16 +20,17 @@
 #
```

Comment:
Use this template for non-init files: https://github.com/OCA/maintainer-tools/blob/c73e7d8ca936cce206256935ec20ad7083b7f176/template/module/__openerp__.py#L1-L3